pkg/controllers/migration/virtualmachine.go

```diff
@@ -94,29 +94,61 @@ func (h *virtualMachineHandler) OnVirtualMachineChange(_ string, vmObj *migratio
 
 	vm := vmObj.DeepCopy()
 	switch vm.Status.Status {
-	case "": // run preflight checks and make vm ready for import
+	case "":
+		// run preflight checks and make vm ready for import
+		logrus.WithFields(logrus.Fields{
+			"name":                    vm.Name,
+			"namespace":               vm.Namespace,
+			"spec.virtualMachineName": vm.Spec.VirtualMachineName,
+		}).Info("Running preflight checks ...")
 		return h.reconcilePreFlightChecks(vm)
-	case migration.SourceReady: //vm migration is valid and ready. trigger migration specific import
+	case migration.SourceReady:
+		// vm migration is valid and ready. trigger migration specific import
+		logrus.WithFields(logrus.Fields{
+			"name":                    vm.Name,
+			"namespace":               vm.Namespace,
+			"spec.virtualMachineName": vm.Spec.VirtualMachineName,
+		}).Info("Importing client disk images ...")
 		return h.runVirtualMachineExport(vm)
-	case migration.DisksExported: // prepare and add routes for disks to be used for VirtualMachineImage CRD
+	case migration.DisksExported:
+		// prepare and add routes for disks to be used for VirtualMachineImage CRD
+		logrus.WithFields(logrus.Fields{
+			"name":                    vm.Name,
+			"namespace":               vm.Namespace,
+			"spec.virtualMachineName": vm.Spec.VirtualMachineName,
+		}).Info("Creating VM images ...")
 		return h.reconcileDiskImageStatus(vm)
 	case migration.DiskImagesSubmitted:
 		// check and update disk image status based on VirtualMachineImage watches
+		logrus.WithFields(logrus.Fields{
+			"name":                    vm.Name,
+			"namespace":               vm.Namespace,
+			"spec.virtualMachineName": vm.Spec.VirtualMachineName,
+		}).Info("Evaluating VM images ...")
 		err := h.reconcileVMIStatus(vm)
 		if err != nil {
 			return vm, err
 		}
-
 		newStatus := evaluateDiskImportStatus(vm.Status.DiskImportStatus)
 		if newStatus == nil {
 			return vm, nil
 		}
 		vm.Status.Status = *newStatus
 		return h.importVM.UpdateStatus(vm)
 	case migration.DiskImagesFailed:
+		logrus.WithFields(logrus.Fields{
+			"name":                    vm.Name,
+			"namespace":               vm.Namespace,
+			"spec.virtualMachineName": vm.Spec.VirtualMachineName,
+		}).Error("Failed to import client disk images. Try again ...")
 		return h.triggerResubmit(vm)
 	case migration.DiskImagesReady:
 		// create VM to use the VirtualMachineObject
+		logrus.WithFields(logrus.Fields{
```

Comment:
may be message needs to be "checking Disk images"?

Reply:
This is the message of the `DiskImagesSubmitted` stage, see some lines above. In the `DiskImagesReady` stage the function `h.createVirtualMachine(vm)` is called to create the virtual machines.

Have you perhaps slipped a few lines with your comment? Do you prefer `Checking disk images` over `Evaluating VM images`?

Reply:
you are correct. my bad 🤦